src/init/features/dataconnect/sdk.ts

```diff
@@ -142,12 +147,15 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
   }
 
   if (targetPlatform === Platform.ANDROID) {
+    // app/src/main is a common practice for Andorid, but not explicitly required.
+    // If it is present, we'll use it. Otherwise, we fall back to the app directory.
+    const baseDir = fs.existsSync(path.join(appDir, "app/src/main"))
+      ? path.join(appDir, "app/src/main")
+      : appDir;
```

Comment:
Suggest `path.join(appDir, "generated")` instead of `appDir`, then remove the logic to append `"generated"` from line 158 below.